terraform/environments/mlra/locals.tf

```diff
@@ -1,12 +1,71 @@
 #### This file can be used to store locals specific to the member account ####
```

Comment:
All the local vars outside of our modules are now defined here, separate from platform_vars which are managed by MP